smk/binning_preparation.smk

```diff
@@ -623,31 +623,44 @@ rule combine_depth:
     shell:
         """
         if [ "{params.multi}" == "yes" ]; then
-            head -1 {input.depths[0]} > combined.depth
-            (for file in {input.depths}; do tail -n +2 $file; done) >> combined.depth
-            rsync -a combined.depth {output}
+            (bash -c "zcat {input.depths[0]} | head -1";
+             for file in {input.depths}; do zcat $file | tail -n +2; done
+            ) | gzip -c > combined.depth.gz
+            rsync -a combined.depth.gz {output.depth_combined}
         else
-            ln --symbolic --relative --force {input[0]} {output}
+            ln --symbolic --relative --force {input[0]} {output.depth_combined}
         fi
         """
 
```

Comment:
I'm under-caffeinated, but why keep rule combine_depth, if its output is not used in make_abundance_npz afterwards?

Reply:
Probably because I was under-caffeinated. It should be removed.

Reply:
Done now (2fcd13b)